admin/desk/utils.py

```diff
@@ -0,0 +1,194 @@
+"""
+Modified from https://gist.github.com/pamelafox/5855372
+"""
+
+import requests
+import json
+from requests_oauthlib import OAuth1Session
+
+from django.conf import settings
+
+
+class DeskError(Exception):
```

Comment:
I understand the rationale behind subclassing `Exception`. I don't think the overrides are necessary.

```python
In [29]: e = Exception('Verbose description of error')

In [30]: e.__str__()
Out[30]: 'Verbose description of error'

In [31]: e.__unicode__()
Out[31]: u'Verbose description of error'

In [32]: e.message
Out[32]: 'Verbose description of error'
```